src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage,
+  closeError: () => void,
+};
+
+export const Error: React.FC<Props> = ({
+  errorMessage,
+  closeError,
+}) => {
```

Comment:
Consider giving the Error component a more specific name, such as 'ErrorNotification', to avoid confusion with JavaScript's built-in Error class. 😊